### ISSUE TITLE
airgap - rancher/fleet:v0.3.0 is used but rancher/fleet:v0.3.0-rc3 is in the images.txt file

### ISSUE THREAD

**Issue by jiaqiluo** [rancher/rancher#29591]
**What kind of request is this (question/bug/enhancement/feature request):**
bug

**Steps to reproduce (least amount of steps as possible):**
- install rancher `v2.5.2-rc1` in HA mode in airgap environment 

**Result:**
- in the local cluster, the fleet-controller uses the image `rancher/fleet:v0.3.0` which is not in the `images.txt` file, instead, `rancher/fleet:v0.3.0-rc3` is included. 

 
<img width="1481" alt="96323631-c81fad00-0fd2-11eb-871e-79efe240a6a5" src="https://user-images.githubusercontent.com/6218999/96503684-824c2a00-1208-11eb-9cde-353208add5fe.png">

```
> cat rancher-images.txt | grep -i fleet
rancher/fleet-agent:v0.3.0-rc3
rancher/fleet:v0.3.0-rc3
```

Which image should rancher `v2.5.2-rc1` use?

**Comment by mrajashree** [comment]
Can be tested with v2.5.2-rc3

**Comment by jiaqiluo** [comment]
the bug fix is validated in rancher:`v2.5.2-rc3` airgap installation 



![image](https://user-images.githubusercontent.com/6218999/96793961-2d4c1780-13b2-11eb-8e49-b2a0281abe4b.png)

```
> cat rancher-images.txt | grep -i fleet
rancher/fleet-agent:v0.3.0
rancher/fleet:v0.3.0
```
